src/sts.rs

```diff
@@ -0,0 +1,278 @@
+//! The AWS STS API.
+//!
+//! If you're using the service, you're probably looking for [StsClient](struct.StsClient.html).
+#![allow(unused_imports)]
+
+include!(concat!(env!("OUT_DIR"), "/sts.rs"));
+
+pub use self::credential::{
```

Comment:
Are the contents of this file needed because our current codegen makes the client for STS but none of the Rusoto credential required behavior?  I believe that's the reason but would like confirmation so I better understand it.  And if that's the case, let's add a comment in the code to that effect.